packages/aws-cdk-lib/aws-autoscaling/README.md

```diff
@@ -539,10 +539,20 @@ to determine which instances it terminates first during scale-in events. You
 can specify one or more termination policies with the `terminationPolicies`
 property:
 
+[Custom termination policy](https://docs.aws.amazon.com/autoscaling/ec2/userguide/lambda-custom-termination-policy.html) with lambda
+can be used to determine which instances to terminate based on custom logic.
+The custom termination policy can be specified using `TerminationPolicy.CUSTOM_LAMBDA_FUNCTION`. If this is
+specified, you must also supply a value of lambda arn in the `terminationPolicyCustomLambdaFunctionArn` property.
+
+If there are multiple termination policies specified,
```

Comment:
I would also add some note here saying that to allow the lambda function to terminate ASG, it would needs appropriate permission, like example provided [here](https://docs.aws.amazon.com/autoscaling/ec2/userguide/lambda-custom-termination-policy.html#lambda-custom-termination-policy-create-function)

Reply:
Updated in the latest build